Jint/Runtime/Interop/DefaultTypeConverter.cs

```diff
@@ -256,18 +256,43 @@ private Delegate BuildDelegate(Type type, Func<JsValue, JsValue[], JsValue> func
                 parameters[i] = Expression.Parameter(arguments[i].ParameterType, arguments[i].Name);
             }
 
-            var initializers = new MethodCallExpression[parameters.Length];
-            for (var i = 0; i < parameters.Length; i++)
+            MethodCallExpression[] initializers;
+
+            if (arguments.Length == 1 &&
```

Comment:
now this limits to having single parameter in function declaration, if you look how `MethodInfoFunctionInstance.Call` handles this it checks for each parameter. I'm unsure if this can easily handle same logic or should this somehow use the same logic.

Reply:
Changed. There are no limits now. If the use of ```List``` is not allowed, I can change it to an array querying before the number of parameters the ```FunctionInstance``` expects.